packages/mui-material/src/Hidden/withWidth.js

```diff
@@ -108,8 +107,6 @@ const withWidth =
       WithWidth.displayName = `WithWidth(${getDisplayName(Component)})`;
     }
 
-    hoistNonReactStatics(WithWidth, Component);
```

Comment:
For context, `Component` can only be Hidden internal components, so this line is useless.